src/playdate/domain/lcdbitmap.nim

```diff
@@ -0,0 +1,40 @@
+import std/importutils
+import ../bindings/[api, types, graphics]
```

Comment:
This seems like a lot of imports for a domain model. Nim forces my hand becase the =destroy proc must be declared in the same file as the type

Reply:
I think it's fine